community/contribute/team/index.md

```diff
@@ -0,0 +1,31 @@
+# Team Contributing Guidelines
+
+## Communication across team and collaborators
+
+Our general philosophy is to limit meetings and emails as much as is
+appropriate, and instead use text-based forms of communication. For
+shorter or quick questions, we use our Discord channel. For longer
+discussions, we use GitHub Issues. And for help or sharing information,
+we use blog posts that we add to the website.
+
+No matter which communication channel is used, we write in Markdown
```

Comment:
Should we except Discord here?

Reply:
Well, you can use a limited set of Markdown, but not completely. I'll clarify in the text.